The current version of libdvdcss has removed the dvdcss_interface_2 symbol so
burner incorrectly assumes it's the wrong version.

https://bugzilla.redhat.com/show_bug.cgi?id=1193628
https://bugzilla.gnome.org/show_bug.cgi?id=744916

Index: burner-3.12.1/plugins/dvdcss/burn-dvdcss.c
===================================================================
--- a/plugins/dvdcss/burn-dvdcss.c
+++ b/plugins/dvdcss/burn-dvdcss.c
@@ -95,9 +95,6 @@ burner_dvdcss_library_init (BurnerPlug
 	if (!module)
 		goto error_doesnt_exist;
 
-	if (!g_module_symbol (module, "dvdcss_interface_2", &address))
-		goto error_version;
-
 	if (!g_module_symbol (module, "dvdcss_open", &address))
 		goto error_version;
 	dvdcss_open = address;
